packages/analytics-js-integrations/src/integrations/TVSquared/browser.js

```diff
@@ -21,7 +21,6 @@ class TVSquared {
     this.analytics = analytics;
     this.brandId = config.brandId;
     this.clientId = config.clientId;
-    this.eventWhiteList = config.eventWhiteList || [];
```

Comment:
Where are we storing allow list?

Reply:
The allow list is sdk level feature we are not storing it on destination specific browser.js files.

Reply:
ok